### ISSUE TITLE
JJB 3/ MG21 Dash Bug

### ISSUE THREAD

**Issue by HeartedMine** [vret001/KU#31]
**Describe the bug**
Using 3 as jjb will cause it to go down and forward at the same time instead of just forward (same goes to Mg21's dash)

**When did you last experience the issue:**
19/2/22 2:24pm

**To Reproduce**
JJB 3/ MG21 Dash Bug
1. Spawn as JJB/Mg21
2. use 3/ v
3. See error

**Expected behavior**
It sometimes will cause mg21to be tp back to where it was and damage it as well.

**Desktop (please complete the following information):**
 - OS: Window 10
 - Browser Chrome
 - Version 21H1

**Additional context**
basically happens anytime at anywhere
### ****

**Comment by vret001** [comment]
Part of #32, please followup there.
